src/pages/Search/SearchPage.tsx

```diff
@@ -52,19 +51,23 @@ function SearchPage({route}: SearchPageProps) {
 
     if (shouldUseNarrowLayout) {
         return (
-            <FreezeWrapper>
-                <SearchPageNarrow
-                    queryJSON={queryJSON}
-                    policyID={policyID}
-                    shouldGroupByReports={shouldGroupByReports}
-                    searchName={searchName}
-                />
-            </FreezeWrapper>
+            <SearchPageNarrow
+                queryJSON={queryJSON}
+                policyID={policyID}
+                shouldGroupByReports={shouldGroupByReports}
+                searchName={searchName}
+            />
         );
     }
 
     return (
-        <FreezeWrapper>
+        <ScreenWrapper
+            testID={Search.displayName}
+            shouldShowOfflineIndicatorInWideScreen
+            offlineIndicatorStyle={styles.mtAuto}
+            shouldEnableMaxHeight
+            headerGapStyles={styles.searchHeaderGap}
```

Comment:
We added `styles.searchHeaderGap` but did not remove the `<HeaderGap />` component causing https://github.com/Expensify/App/issues/59194.